cves/CVE-2013-0861.yml

```
@@ -64,7 +64,17 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A memory corruption vulnerabilty in utlis.c for audio conversion allowing 
```

Looks like a typo spelling `utils.c` as `utlis.c`.
```suggestion
  A memory corruption vulnerabilty in utils.c for audio conversion allowing 
```